goosebit/settings/__init__.py

```diff
@@ -0,0 +1,18 @@
+from .const import BASE_DIR, DB_MIGRATIONS_LOC, PWD_CXT, SECRET  # noqa: F401
+from .schema import GooseBitSettings
+
+config = GooseBitSettings()
+print(config)
+
+ARTIFACTS_DIR = config.artifacts_dir
```

Comment:
Does it make sense to keep these constants? Why not just use `config.artifacts_dir` directly in the code?

Reply:
Totally fine with that.  Let me update to that then.